File: SKKU/algorithms/find_sum.c
#include <stdio.h>

float FindSum(float list[], int n)
{  
    float sum = 0;
    int i;
    for (i = 0; i < n; i++)
        sum += list[i];
    return sum;
}

void main() {
    float list[] = {10.5, 4.8, 13.2, 6.4, 9.5};
    int n = 5;
    float sum = FindSum(list, n);

    printf("Sum = %f\n", sum);
}

File: SKKU/algorithms/find_sum.rs


fn find_sum(list:&Vec<f32>) -> f32 {  
    let mut sum:f32 = 0.0;
    for i in 0..list.len() {
        sum += list[i];
    }
    sum
}

fn main() {
    let list:Vec<f32> = 
        vec![10.5, 4.8, 13.2, 6.4, 9.5];
    let sum:f32 = find_sum(&list);

    println!("Sum = {}", sum);
}